tests/integration/tier0/removed-certificate/test_removed_certificate.py

```diff
@@ -0,0 +1,23 @@
+import os.path
```

Comment:
Add some comment saying that for next release when the certificate will be restored during rollback this test has to be changed/removed. With a link to a jira issue.

Reply:
Comment added in 4f82815fbca2c8dc6fbf20197c536b7e1933b061